server/selinux/server/pulp-streamer.te

```diff
@@ -0,0 +1,75 @@
+# When built from RPM, the below version of "0.0.0" will be
+# replaced with the version in the spec file
+policy_module(pulp-streamer, 0.0.0)
+type streamer_t;
+type streamer_exec_t;
+init_daemon_domain(streamer_t, streamer_exec_t);
+
+require {
+        type streamer_t;
+        type proc_t;
+        type celery_exec_t;
+        type squid_t;
+        type rpm_exec_t;
+}
+
+
+##### getattr Files #####
+
+# { getattr } on /usr/bin/celery
+allow streamer_t celery_exec_t:file getattr_file_perms;
+
+# { getattr } on /usr/bin/yum
+allow streamer_t rpm_exec_t:file getattr_file_perms;
+
+
+##### Kernel Related #####
+
+## Read system state information in /proc
+kernel_read_system_state(streamer_t);
```

Comment:
Here are the denials specifically. I'm going to make the comment above make it clear which files are being accessed.

```
Feb 18 21:23:42 f23-lazy-selinux.os1.phx2.redhat.com audit[6834]: AVC avc:  denied  { read } for  pid=6834 comm="pulp_streamer" name="meminfo" dev="proc" ino=4026531989 scontext=system_u:system_r:streamer_t:s0 tcontext=system_u:object_r:proc_t:s0 tclass=file permissive=1
Feb 18 21:23:42 f23-lazy-selinux.os1.phx2.redhat.com audit[6834]: AVC avc:  denied  { open } for  pid=6834 comm="pulp_streamer" path="/proc/meminfo" dev="proc" ino=4026531989 scontext=system_u:system_r:streamer_t:s0 tcontext=system_u:object_r:proc_t:s0 tclass=file permissive=1
Feb 18 21:23:42 f23-lazy-selinux.os1.phx2.redhat.com audit[6834]: AVC avc:  denied  { getattr } for  pid=6834 comm="pulp_streamer" path="/proc/meminfo" dev="proc" ino=4026531989 scontext=system_u:system_r:streamer_t:s0 tcontext=system_u:object_r:proc_t:s0 tclass=file permissive=1
Feb 18 21:23:48 f23-lazy-selinux.os1.phx2.redhat.com audit[6972]: AVC avc:  denied  { read } for  pid=6972 comm="pulp_streamer" name="meminfo" dev="proc" ino=4026531989 scontext=system_u:system_r:streamer_t:s0 tcontext=system_u:object_r:proc_t:s0 tclass=file permissive=1
Feb 18 21:23:48 f23-lazy-selinux.os1.phx2.redhat.com audit[6972]: AVC avc:  denied  { open } for  pid=6972 comm="pulp_streamer" path="/proc/meminfo" dev="proc" ino=4026531989 scontext=system_u:system_r:streamer_t:s0 tcontext=system_u:object_r:proc_t:s0 tclass=file permissive=1
Feb 18 21:23:48 f23-lazy-selinux.os1.phx2.redhat.com audit[6972]: AVC avc:  denied  { getattr } for  pid=6972 comm="pulp_streamer" path="/proc/meminfo" dev="proc" ino=4026531989 scontext=system_u:system_r:streamer_t:s0 tcontext=system_u:object_r:proc_t:s0 tclass=file permissive=1
```